build.gradle

```diff
@@ -26,7 +26,7 @@ ext {
   javaVersion = JavaVersion.VERSION_1_7
 
   GROUP = 'com.squareup.leakcanary'
-  VERSION_NAME = "1.4-SNAPSHOT"
+  VERSION_NAME = "1.4-beta2"
```

Comment:
Nope. Master should always be in snapshot, ie here still be 1.4-SNAPSHOT. This PR needs an extra commit that changes `1.4-beta2` to `1.4-SNAPSHOT`